tests/libtest/codechecker.py

```diff
@@ -405,19 +408,20 @@ def remove_test_package_product(test_folder, check_env=None):
         server_data['check_env'] = check_env
 
     # Clean the previous session if any exists.
-    logout(server_data, test_folder)
+    logout(server_data, test_folder, protocol)
 
+    url = protocol + "://"+server_data['viewer_host'] +\
```

Comment:
Maybe you could use the just introduced util here?

Reply:
Please format the code better so `+` is wrapped in spaces around the operand.